src/utils.ts

```diff
@@ -0,0 +1,90 @@
+import * as vscode from 'vscode';
+
+export async function runJar(jarname: string) {
+    // Find .jar files recursively in each workspace and in all directories
+    // Run .jar with currently open file
+    const workspaceFolders = vscode.workspace.workspaceFolders?.map(folder => folder.uri) || [];
+    if (workspaceFolders.length == 0) {
+        vscode.window.showErrorMessage("No workspace folder opened!");
+    } else {
+        let foundJar;
+        for (const folder of workspaceFolders) {
+            foundJar = await findJarAndRun(folder, jarname);
+        }
+        if (!foundJar) {
+            vscode.window.showErrorMessage(`Unable to find ${jarname}.jar`)
```

Comment:
```suggestion
            vscode.window.showErrorMessage(`Unable to find ${jarname}.jar. Please ensure that the ${jarname}.jar file exists somewhere in the current workspace.`)
```

The purpose of this is just to provide a more informative error message.